### PR TITLE
Add ganggo project to info page

### PR DESCRIPTION
**Pull request by Zauberstuhl**
see https://github.com/ganggo

### CODE REVIEW

File: src/texts.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ var texts = {
       website: 'hubzilla.org',
       website_protocol: 'https',
       github: 'https://github.com/redmatrix/hubzilla'
+    },
+    ganggo: {
+      title: 'GangGo',
+      subtitle: 'A Decentralized Social Network',
+      description: 'Open source, decentralized and privacy oriented social network written in GoLang',
+      website: 'https://github.com/ganggo',
```
Comment (jaywink) [inline src/texts.js]: This is silly, but I just realized that this needs a little tunage. `website` should be without protocol like the others so just `github.com/ganggo` - the protocol part is added separately. I don't remember the reason for this - possibly to display it without protocol and not have to remove protocol, which I know would be super hard ;)

**Comment by jaywink** [comment]
Also, an unrelated note, you should fix your `local posts` and `local comments` to only count local posts and comments. Currently it looks like every post (and comment?) is included - which might look like GangGo is very active :D

selection_361

**Comment by Zauberstuhl** [comment]
> which might look like GangGo is very active :D

it will be ;) but yes you are right.. I implemented nodeinfo with you while being drunk. I am surprised that it works at all ^^

**Comment by Zauberstuhl** [comment]
wuhuuuhuu 🍻

**Pull request merged**